lib/util/exec/index.ts

```diff
@@ -136,6 +136,16 @@ async function prepareRawExec(
     };
   }
 
+  if (
+    GlobalConfig.get('binarySource') === 'global' &&
+    opts.toolConstraints?.length
+  ) {
+    logger.once.debug(
+      { toolConstraints: opts.toolConstraints },
+      'When running in binarySource=global, any `constraints` are ignored',
```

Comment:
This can also be set by manager, so a user doesn't have a real influence on this message.
```suggestion
      'Ignoring tool contraints because of `binarySource=global`',
```